### DISCUSSION TITLE
Load websight-io inside Iframe window

### DISCUSSION THREAD

**Discussion by skumar84pal** [websight-io/starter#161] [Q&A]
I want to load my currently developed web application inside the Iframe. But, when I am loading it inside the Iframe than it not allowing me to login into the system. It redirects back to the login page while it is working fine without an iframe.

Does anyone face the same issue? How can we fix this issue?

**Comment by paweljozwicki** [comment]
Hi, if I understand the problem correctly, it's not strictly related to WebSight itself, but a common problem with authorization in applications run in iframe. 
It can be related e.g. to Third Party Cookie Blocking Policy. If so, [Storage Access API](https://webkit.org/blog/8124/introducing-storage-access-api/) should help.
